host/ProtocolGateway.Host.Fabric.FrontEnd/PackageRoot/Config/FrontEnd.Mqtt.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "RetainPropertyName": "mqtt-retain",
+  "DupPropertyName": "mqtt-dup",
+  "QoSPropertyName": "mqtt-qos",
+  "MqttInboundTemplates": [ "devices/{deviceId}/messages/events" ],
+  "MqttOutboundTemplates": [ "devices/{deviceId}/messages/devicebound" ],
+  "MqttQoSStateProvider": "AzureStorage",
+  "ConnectArrivalTimeout": "00:00:00",
+  "MaxPendingInboundAcknowledgements": "16",
+  "DeviceReceiveAckTimeout": "00:00:00",
```

Comment:
>00:00:00 [](start = 30, length = 8)

let's keep configurations in sync across host options. For this specifically, I'm fine making it 00:00:00 across the board.